doc/dev/developer_guide/debugging-gdb.rst

```diff
@@ -0,0 +1,32 @@
+GDB - The GNU Project Debugger
+==============================
+
+`The GNU Project Debugger (GDB) <https://www.sourceware.org/gdb>`_ is
+a powerful tool which allows you to analyze the execution flow.
+GDB can help to find bugs, uncover crash errors or track the
+source code during execution of a development cluster.
+GDB can also be used to debug Teuthology tests runs.
```

Comment:
Again, very picky suggestion
```suggestion
GDB can help to find bugs, uncover crash errors, or track the
source code during execution of a development cluster.
It can also be used to debug Teuthology tests runs.
```